scripts/build/Platform/Windows/build_config.json

```diff
@@ -36,6 +36,18 @@
       "test_cpu_profile_vs2019"
     ]
   },
+  "aws_integration_deployment": {
+    "TAGS": [
+      "nightly-aws-integration-deployment"
+    ],
+    "COMMAND": "deploy_cdk_applications.cmd",
+    "PARAMETERS": {
+      "O3DE_AWS_PROJECT_NAME": "AWSAUTO",
+      "O3DE_AWS_DEPLOY_REGION": "us-west-2",
+      "ASSUME_ROLE_ARN": "arn:aws:iam::645075835648:role/o3de-automation-tests",
```

Comment:
It might be better to parameterize this ARN/Region and move it to a Jenkins envvar. If you move between production and sandbox Jenkins, you want some flexibility to have different environment account and roles.

You can refer to envvars with bangs like this
```
      "O3DE_AWS_DEPLOY_REGION": "!AWSI_DEPLOY_REGION!",
      "ASSUME_ROLE_ARN": "!AWSI_DEPLOY_ROLE_ARN!",
```

I can to work with you on the naming convention if you want to do this.

Reply:
Sure. Will sync offline to set the envvar.

Reply:
+1 We should avoid hard-coding ARNs for specific AWS resources. This will be different depending if we run this in our fork pipeline or in the O3DE pipeline which are running in separate AWS accounts. This also prevents maintainers that fork O3DE from running these tests in their account.

Reply:
To move these parameters to Jenkins envvar, it might be better to create a new Jenkinsfile instead of adding them to the Jenkinsfile used by AR builds.

Reply:
Just updated the PR to use envvar. Currently, this job just deploys the CDK applications and will not test the engine. However, the CDK application code are in the main repo and we don't have a separate awsi repo for them. Also we talked about the use case for GameLift during the office hour. After we add the automation tests for GameLift gem, we do want to build the dedicated server and upload it with this new Jenkins job. In that case, it needs the engine code.

Reply:
Talked with Mike and Shirang offline. We will create a separate jenkinsfile pipeline for the resource deployment instead.